eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py

```diff
@@ -145,101 +134,96 @@ def validate_evidence_string(ev_string, clinvar_record, trait, ensembl_gene_id,
     try:
         ev_string.validate(ot_schema_contents)
     except jsonschema.exceptions.ValidationError as err:
```

Comment:
Any reason the error messages are not written to stderr (through logger or other mechanisms)?

Reply:
No particular reason, it's just a very old code fragment which survived all refactoring iterations :) Now fixed: https://github.com/EBIvariation/eva-opentargets/pull/167/commits/7b455c2a2b53219920b7c691eabe2c3d3971ff2c